API.md

```diff
@@ -24,79 +23,36 @@
 
 Creates a singleton instance of __toki__ and starts the bootstrapping process.
 
-- `options` - required object.
++ `options` __required__ At minimum, a router is required. In most cases you only need to initialize the bridge itself, which in turn will setup Toki for you. Toki accepts the following:
 
-    - `router` - required __toki-bridge__ router object to be setup with configured routes.
-
-
-```javascript
-const bridge = require('toki-hapi-bridge');
-const Toki = require('toki');
-const toki = new Toki({
-    router : bridge
-});
-```
-
-### getInstance()
-[static method](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes/static) - If a __toki__ instance was already created with `new Toki` it will return that instance otherwise will throw an error.
-
-happy path:
-
-```javascript
-const bridge = require('toki-hapi-bridge');
-const Toki = require('toki');
-const toki = new Toki({
-    router : bridge
-});
-
-//rejoice
-const sameTokiInstance = Toki.getInstance();
-
-```
-
-not so happy here:
-
-```javascript
-const Toki = require('toki');
-
-//tears rolling down my face ater error gets thrown
-const notMyToki = Toki.getInstance();
-
-```
+    + `router` A configured bridge
+    + `logger` An optional logger to make use of. Should expose log4j style methods (`error`, `warn`, `info`, `debug`, `trace`)
 
 
 ### events
 
 #### ready
 
-Once __toki__ has been instantiated it will immediately require __toki-config__ to retrieve the configuration and prepare itself to configure routes and action handlers.
+Fired when __toki__ is ready.
 
 ```Javascript
 //intantiate toki
 const Toki = require('toki');
-const toki = new Toki({
-    router : routerInstance
-});
 
-//wait for toki to be ready
 toki.on('ready', ()=>{
-    //toki is open for buisness (aka do your thing)    
+    //Ready  
```

Comment:
Toki is where fun goes to die. All work and no play makes Toki efficient and brutal. :P